.github/workflows/CI.yml

```
@@ -43,29 +47,31 @@ jobs:
 
       - id: cache-rust
         name: cache rust
-        uses: Swatinem/rust-cache@v2
+        uses: Swatinem/rust-cache@c19371144df3bb44fab255c43d04cbc2ab54d1c4 # v2.9.1
         with:
           key: v3
 
       - name: set up python
-        uses: actions/setup-python@v6
+        uses: actions/setup-python@a309ff8b426b58ec0e2a45f0f869d46889d02405 # v6.2.0
         if: "${{ !matrix.disable-gil }}"
```

actionlint (1.7.12)

[error] property "disable-gil" is not defined in object type {pandas-version: string; python-version: string}

(expression)